package.json

```diff
@@ -28,6 +28,11 @@
   },
   "dependencies": {
     "@saritasa/eslint-config-base": "^0.1.0",
-    "eslint-config-prettier": "^2.7.0"
+    "eslint-config-prettier": "^2.7.0",
+    "eslint-plugin-jsx-a11y": "^6.1.1",
+    "eslint-plugin-react": "^7.10.0"
+  },
+  "devDependencies": {
+    "eslint-plugin-flowtype": "^2.50.0"
```

Comment:
Why is not it regular dependency?

Reply:
"@saritasa/eslint-config-base" and "eslint-config-prettier" already were there. I just added next. Should I replace them to dependencies?